stable/kommander/requirements.lock

```diff
@@ -4,10 +4,10 @@ dependencies:
   version: 0.1.12
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.4.8
+  version: 0.5.0
 - name: kommander-licensing
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.4.8
+  version: 0.5.0
 - name: auto-provisioning
```

Comment:
@jimmidyson I was thinking about it. I am wondering if we need to add auto-provisioning here or simply remove it. Kommander should be deployed with auto-provisioning via Konvoy in any way.

Reply:
It should, you're right. Yeah I think we can remove it here. We can install it separately in kind as we will do for e2e testing.

Reply:
Removed.